web/src/services/gql/provider/links/errorLink.ts

```diff
@@ -6,25 +6,31 @@ import { GQLError } from "@reearth/services/state/gqlErrorHandling";
 export default () => {
   const { setErrors } = useSetError();
 
-  return onError(({ graphQLErrors, networkError }) => {
+  return onError(({ graphQLErrors, networkError, operation }) => {
+
+  // 'x-' prefix marks this as a custom (non-standard) header to prevent naming conflicts with official HTTP headers.
+    const skipGlobalError =
+      operation.getContext()?.headers?.["x-skip-global-error"];
```

Comment:
Please define "x-skip-global-error" as an constant and export, so that we can import it wherever use it in api hooks.